src/c/_cffi_backend.c

```diff
@@ -6109,6 +6108,18 @@ static void _my_PyErr_WriteUnraisable(PyObject *t, PyObject *v, PyObject *tb,
     if (extra_error_line == NULL)
         extra_error_line = "";
 
+#if PY_VERSION_HEX >= 0x030D0000
+    if (obj != NULL) {
+        PyErr_FormatUnraisable("Exception ignored %c%s%R%s",
```

Comment:
Maybe it's an opportunity to enhance the error message?

Should it be "Exception ignored ...", "Exception ignored **in** ..." (add `in `) or  "Exception ignored: ..." (add `: `)?